python/knowledges/batch_upload/scan_file_status.py

```
@@ -0,0 +1,211 @@
+        else:
+            print("\n✅ All scanned files are in 'done' status!")
+    
+    def _format_timestamp(self, timestamp):
+        """Format timestamp for display"""
+        if not timestamp:
+            return "Unknown"
+        try:
+            if str(timestamp).isdigit():
+                return datetime.fromtimestamp(int(timestamp) / 1000).strftime('%Y-%m-%d %H:%M:%S')
+            return str(timestamp)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)